pom.xml

```diff
@@ -256,8 +255,8 @@
             <artifactId>value</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
```

Comment:
All classes depend on the `org.slf4j.Logger` class (as they should), so the slf4j-api dependency should remain in here too.